src/routes/settings/__tests__/SettingsScreen.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+// @ts-expect-error
+import mockRNDeviceInfo from 'react-native-device-info/jest/react-native-device-info-mock'
+
+import labels from '../../../constants/labels.json'
+import render from '../../../testing/render'
+import SettingsScreen from '../SettingsScreen'
+
+jest.mock('react-native-device-info', () => mockRNDeviceInfo)
```

Comment:
Since you are just using `getVersion` mocking it manually here is probably easier and makes the ts-expect-error and eslint-disable-next-linte unnecessary:
```suggestion
jest.mock('react-native-device-info', () => ({
  getVersion: jest.fn(() => '2022.6.3')
}))
```